### PR TITLE
Add tech filters to main page

### PR DESCRIPTION
**Pull request by jonton11**
Latest changes:

2/3 of the filters are in React
Organizations are now React components

However...

filters do not work
they are not yet functional
this haiku is bad

we will resume after demo day

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -61,6 +61,7 @@ gem 'kaminari-bootstrap'
 gem "fog"
 gem 'exception_notification'
 gem 'sidekiq'
+gem 'react-rails'
```
Comment (tkbeili) [inline Gemfile:64]: Awesome!

File: app/controllers/organizations_controller.rb [hunk 2]
```diff
@@ -22,10 +22,12 @@ def show
   end
 
   def index
+    @technologies = Technology.all
     if current_user && current_user.admin?
       @organizations = Organization.page(params[:page]).per(ORGANIZATIONS_PER_PAGE)
     else
       @organizations = Organization.published.page(params[:page]).per(ORGANIZATIONS_PER_PAGE)
+      # @organizations = Organization.search(params)
```
Comment (tkbeili) [inline app/controllers/organizations_controller.rb]: Please remove comment

**Comment by tkbeili** [comment]
Please rebase on top of integration. I made some changes to the map display

**Comment by jonton11** [comment]
Rebased and changes pushed to this branch

File: app/assets/javascripts/components/organization.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    return this.props.searchTerm === null || this.props.organization.name.toLowerCase().match(this.props.searchTerm.toLowerCase());
+  },
+  // techMatched: function() {
+  //   return
+  // },
+  teamSizeMatched: function() {
+    var sizeInput = this.props.sizeInput;
+    var techTeamSize = this.props.organization.tech_team_size;
+
+    return this.props.techSizeSearch === null || (sizeInput === "1" && techTeamSize < 25) || (sizeInput === "2" && techTeamSize > 25 && techTeamSize < 50) || (sizeInput === "3" && techTeamSize > 50);
```
Comment (tkbeili) [inline app/assets/javascripts/components/organization.jsx]: can you please put this on multiple lines to make it easier to read?

File: app/views/organizations/index.html.erb [hunk 4]
```diff
@@ -22,25 +22,20 @@ google.maps.event.addDomListener(window, 'load', initialize);
...
   <%= paginate @organizations %>
 </div>
 <div class="container-fluid text-center">
   <%= page_entries_info @organizations %>
 </div>
+
+<!-- Restrict to three tech stacks -->
+<script>
+  $(".chosen-select").chosen({ max_selected_options: 3 });
+</script>
```
Comment (tkbeili) [inline app/views/organizations/index.html.erb]: Please put this in a `.js` file in assets

**Comment by tkbeili** [comment]
@jonton11 Please fix the last two comments I just added and please make a task on Trello for the things that haven't been completed then merge 👍

**Comment by jonton11** [comment]
Refactored `organization.jsx` and created a separate file `maxChosenSelect.js` in assets folder

Trello card has been updated and moved to Backlog.

**Pull request merged**